neural_compressor/torch/__init__.py

```diff
@@ -11,21 +11,3 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
```

Comment:
Why not directly expose the `quantize` function?
Using `from neural_compressor.torch import quantize` is simpler than `from neural_compressor.torch.quantization import quantize`.

Reply:
1. It's not the same usage as `torch`. 
2. It will confuse user when we contain pruning.

Reply:
Understood, thanks.